### ISSUE TITLE
converting h265 10-bit to h264 8bit with nvec encoder fails

### ISSUE THREAD

**Issue by docmcfreckles** [donmelton/other_video_transcoding#73]
when converting 10bit h265 to 8bit h264 with the nvec encoder I get `[h264_nvenc @ 000002b29823b200] No capable devices found`

after some digging on the ffmpeg forums, it seems the pixel format must be changed to be compatible with the nvenc encoder

solution found with help from [this Superuser post](https://superuser.com/a/1381151)

I was able to get the video to encode by adding `-vf format=yuv420p` to the  `other-transcode` generated ffmpeg string

I'm not sure if it's due to me using an extremely old NV chip (GTX 750Ti)
or if this is something that will occur on other setups on newer hardware.

Is this something that needs a patch to be able to use the script to batch process or is it something I can add a flag for?

**Comment by donmelton** [comment]
@docmcfreckles You're correct, changing the pixel format will allow Nvenc to proceed. However, I'm not going to fix that bug at this time. :)

Converting from 10-bit to 8-bit causes horrible color mapping issues. And neither `ffmpeg` nor Nvidia is able to correct those currently. So I would rather the process fail as an encouragement to the user to not do that.

Does that make sense?
